stdlib/public/runtime/ProtocolConformance.cpp

```diff
@@ -28,13 +28,19 @@
 
 using namespace swift;
 
-#if !defined(NDEBUG) && SWIFT_OBJC_INTEROP
+#if !defined(NDEBUG)
+#if SWIFT_OBJC_INTEROP
 #include <objc/runtime.h>
 
 static const char *class_getName(const ClassMetadata* type) {
   return class_getName(
     reinterpret_cast<Class>(const_cast<ClassMetadata*>(type)));
 }
+#else
+static const char *class_getName(const ClassMetadata* type) {
+  return type->getNominalTypeDescriptor()->Name.get();
+}
```

Comment:
Seemed like the least intrusive change to get this code working on non-ObjC interop platforms so I could debug the protocol compliance import from ELF. It’s unfortunate it uses a functoin name from the runtime I supppose.